deploy/roles/r-libs/tasks/main.yml

```diff
@@ -0,0 +1,100 @@
+---
+- name: Install packages required by R packages
+  become: yes
+  become_method: sudo
+  yum:
+    name: "{{ item }}"
+    state: present
+    disable_gpg_check: yes
+  with_items: "{{ r_dependencies }}"
+  tags:
+    - r-update
+  when: install_r_updates
+
+- name: Find any locked directories
+  become: yes
+  become_method: sudo
+  find:
+    paths: "{{ r_system_package_dir }}"
+    pattern: '00LOCK-*'
+    file_type: directory
+  register: locked_dirs
+  tags:
+    - r-update
+  when: install_r_updates
+
+- name: Remove locked directories
+  become: yes
+  become_method: sudo
+  file:
+    path: "{{ item.path }}"
+    state: absent
+  with_items:
+    - "{{ locked_dirs.files }}"
+  tags:
+    - r-update
+
+- name: Update existing R packages
+  become: yes
+  become_method: sudo
+  command: /usr/bin/Rscript --slave --no-save --no-restore-history -e "update.packages(repos=c('http://cran.rstudio.com/'),ask=FALSE,checkBuilt=TRUE);"
+  environment: "{{ r_env_vars }}"
+  async: 2400
+  poll: 60
+  tags:
+    - r-update
+  when: install_r_updates
+
+#--------------------------------------------------------------- RUpdate ^ - R-Libs v
```

Comment:
@lel99999 We can remove this logical separation, as we always want to update packages before installing new ones.  Let's just consider everything to be `r-libs`.